lib/queue_classic/worker.rb

```diff
@@ -161,5 +163,15 @@ def setup_queues(adapter, queue, queues, top_bound)
       end
     end
 
+    def clone_args(job)
+      job[:args].map do |arg|
```

Comment:
`clone` does a shallow copy, so each individual arg wouldn't get copied (only the `arg` array).